docs/ballerina-by-example/examples/file-api/file-api.bal

```diff
@@ -1,58 +1,73 @@
 import ballerina.file;
 
 function main (string[] args) {
-    //Create 'File' struct and open for writing.
-    file:File target = {path:"/tmp/result.txt"};
-    target.open(file:W);
-
-    //Close the file once done.
-    target.close();
+    //Create 'File' struct.
+    string path = "/tmp/result.txt";
+    file:File target = {path:path};
 
     //Check whether the file exists.
-    boolean b = target.exists();
-    println("file exists: " + b);
+    boolean fileExists = target.exists();
+    println(path + " exists: " + fileExists);
+
+    //Creating the file if it does not exist. All 3 potential errors are ignored.
+    if (!fileExists) {
+        var created, _, _ = target.createNewFile();
 
-    //Here's how you can copy a file.
+        if (created) {
+            println("file successfully created: '" + path + "'");
+        }
+    }
+
+    //Here's how you can copy a file. Copying can result in 3 types of errors which are ignored here.
+    //The boolean parameter instructs not to replace if the file already exists.
     file:File source = {path:"/tmp/result.txt"};
     file:File destination = {path:"/tmp/copy.txt"};
-    file:copy(source, destination);
-    println("file copied: /tmp/result.txt to /tmp/copy.txt");
+    _, _, _ = file:copy(source, destination, false);
```

Comment:
This looks ugly. Here we are returning multiple errors. Please look at how other languages do this sort of things.